assets/js/app.js

```diff
@@ -1,21 +1,21 @@
 // We need to import the CSS so that webpack will load it.
 // The MiniCssExtractPlugin is used to separate it out into
 // its own CSS file.
-import css from "../css/app.css"
+import css from "../css/app.css";
 
 // webpack automatically bundles all modules in your
 // entry points. Those entry points can be configured
 // in "webpack.config.js".
 //
 // Import dependencies
 //
-import "phoenix_html"
+import "phoenix_html";
 
-import { Socket } from "phoenix"
-import LiveSocket from "phoenix_live_view"
+import { Socket } from "phoenix";
+import { LiveSocket } from "phoenix_live_view";
```

Comment:
Have to do this otherwise got this issue https://stackoverflow.com/questions/50435253/webpack-imported-module-is-not-a-constructor